services/QuillLMS/engines/evidence/app/models/evidence/prompt_response.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: evidence_prompt_responses
+#
+#  id        :integer          not null, primary key
+#  embedding :vector(1536)     not null
+#  text      :text             not null
+#
+# Indexes
+#
+#  index_evidence_prompt_responses_on_text  (text) UNIQUE
+#
+
+require 'neighbor'
+
+module Evidence
+  class PromptResponse < ApplicationRecord
```

Comment:
I have some questions about the modeling. So this class has no connection to an `Evidence::Prompt` and seems to be a generic storage of text and an embedding. So a few questions:
1. If you'd like to keep this model generic, we might want to change the model name to something generic.
2. Are you adding a connection to an `Evidence::Prompt` in a different model? Or should a `prompt_id` be added here (this may be the simpler route for now)?

Reply:
After discussing with Dan and noting that the lever for adjusting dimension, I think it makes sense to include `prompt_id` with the model since that will some transparency on the use case for these records and possibly for the selection of model (and dimension).